### ISSUE TITLE
Wrong duration for VBR encoded MP3s

### ISSUE THREAD

**Issue by kadai** [wapmorgan/Mp3Info#16]
Greetings,

First of all, thank you for the work you have been doing with this library. I have found it very useful. And while so, I have come across a little bug.

If a MP3 is encoded as VBR with a set quality, the duration of the track is not properly calculated by the class.

For example, I have found that a track that should have a duration of 1:34 (~94 secs) it is reported to last  ~56.89 seconds instead.

This does not affect tracks that are not VBR encoded.

Further looking into the code, I found out that Quality is used to calculate the $samples_per_second in the lines 295 to 297 in MP3Info.php:

```php
if ($this->isVbr && isset($this->vbrProperties['quality'])) {
    $samples_in_second = floor($samples_in_second * $this->vbrProperties['quality'] / 100);
}
```

After doing some tests, I found out that the "time" difference was related to the quality. In my case, the MP3 had a quality of 60, so the time was 60% shorter than expected.

I am using the dev-master version of the library.

Thanks in advance.

**Comment by wapmorgan** [comment]
This way of calculation had a purpose.
Please, provide an example of mp3 to test it

**Comment by kadai** [comment]
I have attached one of the files that gives me issues.

I created this file using LameXP (that uses Lame CLI if I am not wrong) with the next settings:
 - Average quality (ABR)
 - Bit Rate: ~224kbps

Then I tagged the track with Clementine, MP3Tag and TagScanner... and after that I eventually applied normalization.

[test_audio.zip](https://github.com/wapmorgan/Mp3Info/files/4783248/test_audio.zip)

I suspect that maybe it has to do with the fact it uses ABR (A variant of VBR) and probably supplies aditional information to detect it.

**Comment by ghasrfakhri** [comment]
I have exactly same problem. I attached two sample files.

[5f9027d3cc122.zip](https://github.com/wapmorgan/Mp3Info/files/5415831/5f9027d3cc122.zip)
[5f9028549dca7.zip](https://github.com/wapmorgan/Mp3Info/files/5415833/5f9028549dca7.zip)

**Comment by MakamuEvans** [comment]
Any working solution to this?

**Comment by au-lait77** [comment]
anyone a solution? have the same problem that the duration is not correct with VBR.

**Comment by wapmorgan** [comment]
Fixed in 0.0.6 release
